gradle.properties

```diff
@@ -43,6 +43,8 @@ doctor.enableTaskMonitoring=false
 org.gradle.daemon=true
 org.gradle.caching=true
 org.gradle.parallel=true
+org.gradle.configuration-cache=true
+org.gradle.configuration-cache.parallel=true
```

Comment:
Probably we should also enable "configuration-on-demand" by default, but I'm not sure about it as this mode might silently break something. I would propose enabling this property locally and after couple of weeks we will add it here if everything goes okay.

```properties
org.gradle.configureondemand =true
```